<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="samala-z01" name="com.appium.test.pnclatest.App" tests="5" failures="0" timestamp="2021-03-19T22:04:21 IST" time="170.752" errors="1">
  <testcase name="searchconnect" time="0.000" classname="com.appium.test.pnclatest.App">
    <skipped/>
  </testcase> <!-- searchconnect -->
  <system-out/>
  <testcase name="consolelogin" time="0.000" classname="com.appium.test.pnclatest.App">
    <skipped/>
  </testcase> <!-- consolelogin -->
  <system-out/>
  <testcase name="setup" time="22.890" classname="com.appium.test.pnclatest.App"/>
  <system-out/>
  <testcase name="enrollment" time="147.862" classname="com.appium.test.pnclatest.App">
    <error type="java.lang.StringIndexOutOfBoundsException" message="String index out of range: -11">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: -11
at java.lang.String.substring(String.java:1931)
at com.appium.test.pnclatest.App.enrollment(App.java:176)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.appium.test.pnclatest.main.main(main.java:33)
]]>
    </error>
  </testcase> <!-- enrollment -->
  <system-out/>
  <testcase name="uninstallclient" time="0.000" classname="com.appium.test.pnclatest.App">
    <skipped/>
  </testcase> <!-- uninstallclient -->
  <system-out/>
</testsuite> <!-- com.appium.test.pnclatest.App -->
